bankapp/auth.py

```diff
@@ -102,10 +104,13 @@ def login():
         password = request.form['password']
         db = get_db()
         error = None
+        # sql_statement = 'SELECT * FROM user WHERE username = ' + "'" + username + "'"
```

Comment:
Do we need to keep #107? I didn't see any SQL injection vulnerability we insert.

Reply:
I'll fix that